tools/github-actions/release/action.yml

```diff
@@ -7,9 +7,17 @@ inputs:
   target:
     description: 'The branch to target'
     required: true
+  previousVersionsLimit:
+    description: 'Limit of previous versions to consider for the release notes'
+    required: false
+    default: '100'
 runs:
   using: "composite"
   steps:
+    - name: Previous release tag
+      if:  ${{(!contains( inputs.version, '-' )) && endsWith( inputs.version, '.0' )}}
+      shell: bash
+      run: echo "PREVIOUS_VERSION_TAG=$(gh release list --exclude-drafts --exclude-pre-releases --limit ${{ inputs.previousVersionsLimit }} --json tagName --template '[{{range .}}"{{.tagName}}",{{end}}""]{{"\n"}}' | node ./packaged-action/index.cjs ${{ inputs.version }})" >> $GITHUB_ENV
```

Comment:
are you sure the double quotes are kept in the output ?
I got this in local
![image](https://github.com/user-attachments/assets/6ca32f5e-50d7-47ae-bd03-db262c061c96)
![image](https://github.com/user-attachments/assets/cc1730a9-db52-4224-ad96-115a9f8d129c)

Reply:
You forgot the `"` of the `echo` command, please try with the following command:

```bash
echo "PREVIOUS_VERSION_TAG=$(gh release list --exclude-drafts --exclude-pre-releases --limit 100 --json tagName --template '[{{range .}}"{{.tagName}}",{{end}}""]{{"\n"}}' | node ./tools/github-actions/release/packaged-action/index.cjs 11.2.0)"
```

The output (on Mac/Linux) is :

```
PREVIOUS_VERSION_TAG=v11.1.6
```

I would be interested to have an output on Windows, it may be different

Reply:
my bad, I didn't run the `| node` part and got confused, but there is no problem if we pass the output to node before setting the variable